test/appmon/appmon_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/test/kubeobjects/environment"
 	"github.com/Dynatrace/dynatrace-operator/test/kubeobjects/namespace"
 	"github.com/Dynatrace/dynatrace-operator/test/sampleapps"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
It needs to be imported because of side effects (which is registering gcp auth plugin).
It should be done in single top-level place instead of in specific tests file.